addon/model.js

```diff
@@ -127,14 +127,16 @@ var Model = Ember.Object.extend({
         });
 
         var modelIsDirtyAttrs = attributes.map((attr) => attr + "IsDirty");
-
         defineProperty(model, "isNotDirty", computed.not('isDirty'));
 
-        defineProperty(model, "isDirty", computed(function() {
+        var computedArguments = modelIsDirtyAttrs.slice(0);
+        computedArguments.push(function computeModelIsDirty() {
             var modelAttrs = modelIsDirtyAttrs.filter((attr) => get(model, attr) === true);
 
             return modelAttrs.length > 0;
-        }).property("" + modelIsDirtyAttrs));
```

Comment:
We were passing a single string that was comma separated dependent keys.  Instead we want to pass all the dependent keys into the `computed` and the tail of the argument list should be the computed function.